app/services/custom_fields/hierarchy/hierarchical_item_service.rb

```diff
@@ -97,11 +97,18 @@ def move_item(item:, new_parent:)
 
       # Reorder the item along its siblings.
       # @param item [CustomField::Hierarchy::Item] the parent of the node
-      # @param new_sort_order [Integer] the new parent of the node
+      # @param new_sort_order [Integer] the new position of the node
       # @return [Success(CustomField::Hierarchy::Item)]
```

Comment:
To be honest I would remove the comments. At least the return type needs to be fixed.

Reply:
I like rdoc. They express the intention of the developer. Downside is giving maintenance, but that's fine